src/integrate/scarches/config.vsh.yaml

```diff
@@ -0,0 +1,140 @@
+          direction: input
+          required: true
+        - name: "--modality"
+          type: string
+          default: "rna"
+          required: false
+        - name: "--reference"
+          choices: ["HLCA"]
+          alternatives: ["-r"]
+          type: string
+          description: Name of the reference dataset
+          required: false
+        - name: "--unlabeled_category"
+          type: string
+          description: "Value used for unlabeled cells"
+          default: "Unknown"
+          required: false
+        - name: "--labels_key"
+          type: string
+          description: "Name of the column containing cell types"
+          default: "leiden"
+          required: false
+        - name: "--predicted_labels_key"
+          type: string
+          description: Name of the column that will contain cell types predicted by scANVI
+          default: "predicted_labels"
+          required: false
+        - name: "--input_layer"
+          type: string
+          required: false
+          description: "Input layer to use. If None, X is used"
+        - name: "--obs_batch"
+          type: string
+          default: "sample_id"
+          required: false
+          description: Column name discriminating between your batches.
+        - name: "--var_input"
+          type: string
+          required: false
+          description: ".var column containing highly variable genes. By default, do not subset genes."
+    - name: Outputs
+      arguments:
+        - name: "--output"
+          alternatives: ["-o"]
+          type: file
+          description: Output h5mu file.
+          direction: output
+          required: true
+        - name: "--obsm_output"
+          type: string
+          default: "X_integrated_scanvi"
+          required: false
+          description: "In which .obsm slot to store the resulting integrated embedding."
+    - name: "Early stopping arguments"
+      arguments:
+        - name: "--early_stopping"
+          required: false
+          type: boolean
+          description: "Whether to perform early stopping with respect to the validation set."
+        - name: "--early_stopping_monitor"
+          choices: ["elbo_validation", "reconstruction_loss_validation", "kl_local_validation"]
+          default: "elbo_validation"
+          type: string
+          description: "Metric logged during validation set epoch."
+        - name: "--early_stopping_patience"
+          type: integer
+          min: 1
+          default: 45
+          description: "Number of validation epochs with no improvement after which training will be stopped."
+        - name: "--early_stopping_min_delta"
+          min: 0
+          type: double
+          default: 0.0
+          description: "Minimum change in the monitored quantity to qualify as an improvement, 
+                        i.e. an absolute change of less than min_delta, will count as no improvement."
+    - name: "Learning parameters"
```

Comment:
not needed for workflows 1 and 2

Reply:
maybe for fine-tuning?

Reply:
if needed: defaults from scvi component?

Reply:
Removed these parameters, so they will be default from the scvi_tools